docs/5-user-authentication.md

```diff
@@ -99,6 +99,21 @@ export class RcTestAppComponent {
 }
 ```
 
+## Logout users
+
+`logout()` : Removes the Firebase refernece, similar to [firebase-signout](https://firebase.google.com/docs/reference/js/firebase.auth.Auth#signOut)
```

Comment:
Hi @yashmurty , just a small misspell.
You misspelled reference to **refernece**.
Thanks. 😄